apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/template-update-notice/index.js

```diff
@@ -0,0 +1,28 @@
+/* global fullSiteEditing */
+/**
+ * External dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import { render } from '@wordpress/element';
+import { Notice } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+domReady( () => {
+	if ( 'wp_template_part' !== fullSiteEditing.editorPostType ) {
+		return;
+	}
+	const element = document.createElement( 'div' );
+	element.classList.add( 'wp-block-a8c-template-edit-warning' );
+	document.getElementsByClassName( 'edit-post-layout__content' )[ 0 ].prepend( element );
+	render(
+		<Notice isDismissible={ false }>
+			{ __( 'Updates to this block will affect multiple pages on your site' ) }
```

Comment:
I know that this text has been carried over from the designs in the issues. Would it make sense to state that this "will affect **all** of the pages on your site", given that that's the current behavior? cc @shaunandrews

This might also make it easier for users to figure out how to customize header for dynamic pages, which are now not editable in FSE.

Reply:
One more thing - I'm not sure that "block" is the best term for what we are updating here. I'm not sure what would be the best term to use here. One thing that comes to my mind is "template" which might be more future-proof and aligned with core.

Reply:
Have updated it to your wording as I think it is clearer. Will leave open for a bit longer to see if @shaunandrews has anything to add